Gruntfile.js

```diff
@@ -113,9 +127,16 @@ module.exports = function(grunt) {
       seed: {
         command: 'grunt qunit:seed --seed="7x9"'
       },
+      circularObject: {
+        command: 'grunt qunit:circularObject'
+      },
       failAssert: {
         command: 'grunt qunit:failAssert --with-failing'
       },
+      failCircularObject: {
+        command: 'grunt qunit:failCircularObject --with-failing',
+        maxBuffer: 1000 * 1000 * 1000
```

Comment:
I realized this was in fact not necessary, I only added this while trying to understand why my failure test was not working as I expected. I thought ouput was truncated, but it was only truncated in the debug console 🤦‍♂️

Removed it in #196